clang/lib/Sema/SemaExprMember.cpp

```diff
@@ -990,6 +970,18 @@ Sema::BuildMemberReferenceExpr(Expr *BaseExpr, QualType BaseExprType,
                                const Scope *S,
                                bool SuppressQualifierCheck,
                                ActOnMemberAccessExtraArgs *ExtraArgs) {
+  assert(!SS.isInvalid() && "nested-name-specifier cannot be invalid");
+  if (R.wasNotFoundInCurrentInstantiation() ||
+#if 0
```

Comment:
Oh?

Reply:
The inactive branch of the `#if` directive is left over from when `LookupMemberExprInRecord` would call `LookupQualified` directly. Since it now calls `LookupParsedName`, `R.wasNotFoundInCurrentInstantiation()` will be `true` when the _nested-name-specifier_ in the _id-expression_ of the class member access expression is dependent, so this check is no longer necessary. I'll remove it.